.travis.yml

```diff
@@ -4,7 +4,7 @@ python:
 # command to install dependencies
 install:
   - pip install -r requirements.txt
-  - pip install git+https://github.com/econ-ark/HARK.git@master
+  - pip install git+https://github.com/econ-ark/HARK.git@0.10.4
```

Comment:
I’m not sure about this. Users will find this weird as DemARKs won’t be running with the latest HARK.

Reply:
Ah, I see. You have people using the `binder/requirements.txt` file in the install instructions.

I was confused because `econ-ark` is not in the `requirements.txt` file at the top level.